Documentation/ceph-common-issues.md

```diff
@@ -817,3 +818,70 @@ If you still decide to configure an OSD on LVM, please keep the following in min
+...
+csi-cephfsplugin-6cb75                                          0/3     ContainerCreating   0          15h
+csi-cephfsplugin-9whpq                                          0/3     ContainerCreating   0          15h
+csi-cephfsplugin-bpn88                                          0/3     ContainerCreating   0          15h
+csi-cephfsplugin-gd6kk                                          0/3     ContainerCreating   0          15h
+csi-cephfsplugin-hbjkj                                          0/3     ContainerCreating   0          15h
+csi-cephfsplugin-jt48j                                          0/3     ContainerCreating   0          15h
+csi-cephfsplugin-mlj6w                                          0/3     ContainerCreating   0          15h
+csi-cephfsplugin-provisioner-67cdf965c6-764bx                   0/5     ContainerCreating   0          15h
+csi-cephfsplugin-provisioner-67cdf965c6-pq4wm                   0/5     ContainerCreating   0          15h
+csi-cephfsplugin-rx599                                          0/3     ContainerCreating   0          15h
+csi-rbdplugin-9v8kb                                             0/3     ContainerCreating   0          15h
+csi-rbdplugin-bccpt                                             0/3     ContainerCreating   0          15h
+csi-rbdplugin-bqlpc                                             0/3     ContainerCreating   0          15h
+csi-rbdplugin-f2fb9                                             0/3     ContainerCreating   0          15h
+csi-rbdplugin-h8hbc                                             0/3     ContainerCreating   0          15h
+csi-rbdplugin-l2wbz                                             0/3     ContainerCreating   0          15h
+csi-rbdplugin-njtt7                                             0/3     ContainerCreating   0          15h
+csi-rbdplugin-provisioner-78d6f54775-dq47m                      0/6     ContainerCreating   0          15h
+csi-rbdplugin-provisioner-78d6f54775-hc9nb                      0/6     ContainerCreating   0          15h
+csi-rbdplugin-tfn52                                             0/3     ContainerCreating   0          15h
+...
+```
+
+You hit this problem if ownerReference is set to rook-ceph-operator Deployment in the Ceph-CSI driver CRs (rook-ceph.cephfs.csi.ceph.com and rook-ceph.rbd.csi.ceph.com) as follows.
+
+```console
+$ kubectl get csidrivers.storage.k8s.io rook-ceph.rbd.csi.ceph.com -o yaml
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+...
+  ownerReferences:
+  - apiVersion: apps/v1
+    blockOwnerDeletion: false
+    controller: true
+    kind: Deployment
+    name: rook-ceph-operator
+    uid: 65f3c94b-adbd-4604-b01f-70d045a37e3b
+...
+```
+
+### Solution
+
+If you use Rook <= v1.3.10 or Rook <= v1.4.2, please follow the steps below.
+
+1. Delete the existing CSI drivers:
+
+```console
+kubectl delete csidriver rook-ceph.cephfs.csi.ceph.com
```

Comment:
the driver name depends on the namespace name where the rook operator is deployed

Reply:
If the user has only deployed cephfs or rbd they only need to delete that particular CSI driver